### PR TITLE
react-native: adding visible-password keyboardType

### PR DESCRIPTION
**Pull request by nelyousfi**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

@tdreyno @kamal @leemhenson @farcaller @myabc

### CODE REVIEW

File: types/react-native/index.d.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@
 //                 Kyle Roach <https://github.com/iRoachie>
 //                 Tim Wang <https://github.com/timwangdev>
 //                 Kamal Mahyuddin <https://github.com/kamal>
+//                 Naoufal El Yousfi <https://github.com/nelyousfi>
```
Comment (alloy) [inline types/react-native/index.d.ts]: Just want to make sure that you understand that adding your name here means you’re signing up as maintainer and will get mentions on all future issues and PRs for the react-native typings. Is that what you intended?

**Comment by alloy** [comment]
Change looks good to me, I just wanna know for sure that you wanna sign up as a maintainer.

**Comment by nelyousfi** [comment]
Yes of course. But if I want to undo this in the future, should I just create a new PR without my name?

**Comment by alloy** [comment]
Yup, you can always remove yourself. Cool, thanks for stepping up 👌

**Pull request merged**